src/ValueObjects/Receipt.php

```diff
@@ -41,7 +41,7 @@ final class Receipt
     private $inApp;
 
     /**
-     * @var string
+     * @var string|null
```

Comment:
The [getter method](https://github.com/imdhemy/appstore-iap/blob/main/src/ValueObjects/Receipt.php#L166) needs to be updated as well.